index.html

```diff
@@ -1,246 +1,260 @@
+
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+
+                    </ul>
+                    <div class="about_img about_img_1"></div>
                 </div>
-            </section>
+                <div class="about_row about_row_2">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Frameworks </li>
 
-            <section  class="about_section">
-                <div class="project_text text_color text_header_font">
-                    <h2>About</h2>
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+                    </ul>
+                    <div class="about_img about_img_2"></div>
                 </div>
-                <div class="text_subject text_color text_body_font about_text">
-                    <p>
-                        Hello!I am a software developer! I can 
-                        help you build a product, feature or website.
-                        Take a look of my works. If you like what you see 
-                        and have a project you need coded, don't hesitate 
-                        and contact me.
-                    </p>
+                <div class="about_row about_row_3">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Skills </li>
+                        <li class="li_class"> Github</li>
+                        <li class="li_class"> Terminal</li>
+                    </ul>
+                    <div class="about_img about_img_3"></div>
                 </div>
+            </div>
+
+        </section>
 
-                <a class="text_button text_body_font about_button">
-                    <p> Get my CV</p>
-                    <div class="download_image"></div>  
+        <!-- foooter -->
+        <footer class="contact_section">
+            <div class="mobile_logo"></div>
+
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see
+                and have a project you need coded, don't hesitate
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+  <input
+    type="text"
+    name="FullName"
+    placeholder="Enter Your Name"
+    maxlength="30"
+    required/>
+
+    <input type="email" name="Email" placeholder="Enter Your Email" required />
+    <textarea
+        name="Message"
+        placeholder="Enter Your Message"
+        maxlength="500"
+        required
+    ></textarea>
+    <button class="" type="submit" name="submit-btn">Start collaboration</button>
+    </form>

```

Comment:
- [ ] Please decrease the padding size for the inputs to match the Figma design as `60px` is too big on mobile view.

- [ ] Kindly remove the extra space on the left of the design, as requested by the previous reviewer. This is caused by elements taking more space than they should due to fixed widths.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/160948447-d814bc33-ea11-418f-b20f-acb21d63591a.png) | ![image](https://user-images.githubusercontent.com/62290545/160948480-0748b128-d7e5-473f-821d-8cfc7af455dc.png) |

- You can fix that by doing the following:
   - giving this `h1` tag some font size as its font size gets overwritten by the browser's default font size which is too big for the mobile display.
   https://github.com/principles31/portfolio/blob/3b76f4d78bce5580023534da9452d73e917f95a8/index.html#L37

   - Removing the padding from this class and adjust the width to be in percentage instead of fixed.
   https://github.com/principles31/portfolio/blob/3b76f4d78bce5580023534da9452d73e917f95a8/css/index.css#L105-L107